scripts/commands/updateApiDocs.ts

```diff
@@ -223,14 +200,12 @@ async function convertHtmlToMarkdown(
     },
   );
 
-  const ignore = pkg.ignore ?? (() => false);
-
   let results: Array<HtmlToMdResult & { url: string }> = [];
   for (const file of files) {
     const html = await readFile(join(htmlPath, file), "utf-8");
     const result = await sphinxHtmlToMarkdown({
       html,
-      url: `${pkg.baseUrl}/${file}`,
+      url: `http://localhost:8000/${file}`,
```

Comment:
Great job @arnaucasau realizing that we still need to set `localhost:8000` for the image code to work. But note that we don't actually ever use that URL! I will simplify this code in a follow up PR -- the simplification impacts too many lines of code.